templates/footer.twig

```diff
@@ -1,5 +1,5 @@
 <footer id="footer" class="site-footer site-footer--{{ nav_type }}">
-	{% if new_design_country_selector and nav_type != 'minimal' %}
+	{% if new_design_country_selector %}
```

Comment:
When I was almost done I actually realized that we don't allow to choose any custom colors unless the navigation type is minimal. In which case the country selector is not displayed.

So for campaign themes we don't need to add this variable yet. But it's working already anyway so I just removed this check for now to demonstrate.